test/unit/Makefile

```diff
@@ -3,6 +3,7 @@ all:
 	bapbuild -r -Is analysis,checkers,utils cwe_checker_unit_tests.plugin -pkgs alcotest,yojson,unix,ppx_jane,cwe_checker_core
 	bapbundle install cwe_checker_unit_tests.plugin
 	bap ../artificial_samples/build/arrays_x64_gcc.out --pass=cwe-checker-unit-tests
+	bap ../artificial_samples/build/arrays_x64_clang.out --pass=cwe-checker-unit-tests
```

Comment:
This causes *all* unit tests to run twice. I don't think that this is a good idea.

Reply:
So we just run them with one file as before?

Reply:
That would be my suggestion. Most of the current unit tests do not depend on the file and for those that do we have to find a better solution anyway.